modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -457,8 +465,8 @@ public Response muteDevice(@HeaderParam("Accept") String acceptHeader,
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Path("install-application")
```

Comment:
As we are ideally creating REST resources, better to use a noun for the path rather than a verb.